contents/blog/ab-testing-mistakes.md

```diff
@@ -0,0 +1,99 @@
+---
+title: 5 A/B testing mistakes and how to avoid them
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: 2023-04-24
+featuredImage: todo
+tags: ['experimentation']
+---
+
+Making a mistake in an A/B test can have disastrous implications. At best, it means you've wasted weeks waiting for results that are not useful. At worst, it can lead you to make the wrong product decisions. This blog post will explore five common A/B testing mistakes to avoid to ensure successful experiments!
```

Comment:
To be more realistic, at best nothing happens. Worst case can be very bad though. I'm sure there are examples of A/B tests that have cost companies tons.

Reply:
> To be more realistic, at best nothing happens. 

I disagree :) From my experience, time wasted is more accurate

>  I'm sure there are examples of A/B tests that have cost companies tons.

Interesting idea. I looked into adding an example here of a costly mistake, but after adding it I feel its better to just dive straight into the blog